pkg/crd/schema.go

```diff
@@ -389,8 +389,9 @@ func structToSchema(ctx *schemaContext, structType *ast.StructType) *v1beta1.JSO
 }
 
 // builtinToType converts builtin basic types to their equivalent JSON schema form.
-// It *only* handles types allowed by the kubernetes API standards, meaning
-// no floats (technically we shouldn't allow int64 or plain int either, but :shrug:).
+// It *only* handles types allowed by the kubernetes API standards. Numbers can be
+// int32 and int64. Floats are not allowed. (technically we shouldn't allow
+// int64 or plain int either, but :shrug:).
```

Comment:
you can drop the `int64` from the list of `shouldn't allow`

Reply:
I think this is more along the lines of what you're thinking. And thanks for the direction